### PR TITLE
Kanban project

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
### Kanban project
In this project we:

- Update the readme file
- Add tests
- Add comment popup 
- Add webpack
- Used APIs
- Fix all the required changes

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Jest
+
+
+## Live Demo
+You can view the live demo here
```
Comment (mikenath223) [inline README.md:13]: - [x] Kindly deploy the project and include the live deploy link here.

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
+  <title>Tv Show</title>
+<script defer src="main.js"></script></head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <!-- <div class="logo">
+        <a href="#" class="links">NETFLEX</a>
+      </div> -->
```
Comment (mikenath223) [inline dist/index.html]: - [x] Please remove the commented code here.

File: server.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+//   console.log('Example app listening on port 3000!\n');
```
Comment (mikenath223) [inline server.js]: - [x] Remove the commented code here too.

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
+  <title>Tv Show</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <!-- <div class="logo">
+        <a href="#" class="links">NETFLEX</a>
+      </div> -->
```
Comment (mikenath223) [inline src/index.html]: - [x] Remove the commented code here.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// import './style.css';
+// import 'bootstrap';
+// import Pictures from './api/getMovies.js';
+// import displayPictures from './api/diplayMovies.js';
+
+// const upcoming = new Pictures('upcoming');
+// upcoming.getData().then(() => {
+//   displayPictures(upcoming.data, 'upcoming');
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove the commented code here too.

File: dist/index.html (hunk 2 above)
Comment (Isabellanamutenda) [reply dist/index.html]: Resolved

File: server.js (hunk 3 above)
Comment (Isabellanamutenda) [reply server.js]: Resolved

File: src/index.html (hunk 4 above)
Comment (Isabellanamutenda) [reply src/index.html]: Resolved

File: src/index.js (hunk 5 above)
Comment (Isabellanamutenda) [reply src/index.js]: Resolved

File: README.md (hunk 1 above)
Comment (Isabellanamutenda) [reply README.md:13]: Resolved

Comment (descholar-ceo) [reply README.md:13]: - [ ]  [ optional] This live demo link seems to not be working properly (it is currently showing a white page), would you address that?
screencapture-isabellanamutenda-github-io-Kanban-project-2021-09-17-20_22_02

**Review by descholar-ceo — APPROVED** [review]
Hi @Isabellanamutenda and @codecaiine , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] I am approving this project as the live demo link was not a hard requirement to stop this project from being approved, but before releasing it, please make sure that you fix the live demo link as I mentioned it below

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**